website/docs/docs-section/introduction.md

```diff
@@ -4,31 +4,33 @@ title: Introduction to OpenAttestation
 sidebar_label: Introduction
 ---
 
-OpenAttestation is an open source framework for verifiable document and transferable records.
+OpenAttestation is an open source framework for verifiable documents and transferable records.
 
-## Verifiable Document
+## Verifiable document
 
-Verifiable Document (or verifiable credential) is a tamper-evident document that cryptographically proves who issued it. They are the electronic equivalent of the physical documents that we all possess today, such as: plastic cards, passports, driving licences, qualifications and awards, etc
+A verifiable document (or verifiable credential) is a tamper-evident document that cryptographically proves who issued it. They are the electronic equivalent of the physical documents, such as plastic cards, passports, driving licenses, qualifications and awards, etc.
 
-## Transferable Records
+## Transferable records
 
-Transferable Records are Verifiable Documents which have been extended to define a method for resolution of the owner at any point in time. These records reference properties listed in [the UNCITRAL Model Law on Electronic Transferable Records](https://uncitral.un.org/en/texts/ecommerce/modellaw/electronic_transferable_records). Currently, our only implementation of this is based on [ERC721 NFTs](https://eips.ethereum.org/EIPS/eip-721).
+Transferable records are extended verifiable documents that define a method for identifying the owner of the record at any given time. These records reference properties listed in [the UNCITRAL Model Law on Electronic Transferable Records](https://uncitral.un.org/en/texts/ecommerce/modellaw/electronic_transferable_records). Currently, the OA framework's implementation of this is based on [ERC721 NFTs](https://eips.ethereum.org/EIPS/eip-721).
 
-This allows Transferable Records to be used for documents such as:
+This enables the transferable records to be used for documents such as:
 
 - Title Deeds
 - Bills of Lading
 
 ## Building with OpenAttestation
 
+There are three types of roles who build documents, solutions, or services with the OpenAttestation framework.
+
 ### Integrator
 
-> I want to create OpenAttestation documents or transferable records
+The integrator creates OpenAttestation documents or transferable records.
```

Comment:
Small issue since we are going to restructure the sections. But this is supposed to frame the intentions/desires of each role so it would be more appropriate to say something like
```suggestion
An integrator intends to issue and/or verify OpenAttestation documents or transferable records.
```
and so on

Reply:
updated the role of an integrator, a developer, and a contributor